.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-
+# use [ heartchain/.GitHub ]
```

Comment:
what is this code doing ? also this is commented

Reply:
It was supposed to be <write-delete> aka
<Add(remove)>
Sorry please cancel pull